app/models/scholarship_application.rb

```diff
@@ -0,0 +1,15 @@
+class ScholarshipApplication < ApplicationRecord
+  validate :user_is_verified
+
+  belongs_to :user
+  belongs_to :scholarship
+
+  delegate :name, to: :user
```

Comment:
Perhaps add the `prefix: true` option to these two delegate methods, to be more explicit about these two methods' functionality?  

For example:
```ruby
delegate :name, to: :user, prefix: true
```

That way you would call `scholarship_application.user_name` instead of `scholarship_application.name`.